zebra-network/src/peer_set/initialize.rs

```diff
@@ -101,6 +103,23 @@ where
     ));
 
     // 2. Incoming peer connections, via a listener.
+
+    // Warn if we're configured using the wrong network port.
+    // TODO: use the right port if the port is unspecified
+    //       split the address and port configs?
```

Comment:
I don't think the port can ever be unspecified, since we use a `SocketAddr` in the configuration (in order to avoid exactly this issue).

Reply:
I wanted to make zebrad automatically use the right ports for mainnet and testnet. (At the moment, we default to the mainnet port 8233 regardless of the configured network.)

But if that complicates the design, I think it's ok to default to mainnet, have a warning, and expect testnet users to do some manual configuration.